src/en/rizzcomic/src/eu/kanade/tachiyomi/extension/en/rizzcomic/RizzComic.kt

```diff
@@ -123,15 +108,24 @@ class RealmOasis : MangaThemesia(
         val serialization: String? = null,
         @SerialName("genre_id") val genres: String? = null,
     ) {
+        val slug get() = title.trim().lowercase()
+            .replace(slugRegex, "-")
+            .replace("-s-", "s-")
+            .replace("-ll-", "ll-")
+
         val genreIds get() = genres?.split(",")?.map(String::trim)
+
+        companion object {
+            private val slugRegex = Regex("""[^a-z0-9]+""")
+        }
     }
 
     override fun searchMangaParse(response: Response): MangasPage {
         val result = response.parseAs<List<Comic>>()
 
         val entries = result.map { comic ->
             SManga.create().apply {
-                url = comic.id
+                url = "$mangaUrlDirectory/${comic.slug}/"
```

Comment:
```suggestion
                url = "$mangaUrlDirectory/${comic.slug}/#${comic.id}"
```
keep id in url so no further migration is needed if they swap back to using the pseudo random urls